src/schema/schema_create.c

```diff
@@ -254,6 +254,7 @@ __create_file(WT_SESSION_IMPL *session, const char *uri, bool exclusive, const c
                   uri);
             }
         }
+        WT_STAT_CONN_INCR(session, tiered_files_imported);
```

Comment:
We should rename the variable because it's not related to tiered storage in any way. Maybe `files_imported`?

Reply:
Right, thanks for pointing that out. I'm now wondering where might be the best place to put this statistic since I originally placed it with the other tiered statistics. I'm thinking of putting it with the block manager related statistics but also not sure if there's a better suited place.

Reply:
How about with `SessionOpStat` stats?  Many of them are bigger schema operations, like `table_rename` or `table_drop`, so it seems appropriate.  All those have a success and fail stat, maybe this should do that too?
`session_file_import_success`...

Reply:
I agree with Don, but will put out a few naming possibilities for consideration. All the other operations are `table_<operation>`. An import is part of create. You could name it `session_table_create_import_fail|success`. That is getting lengthy but also keeps it alphabetized with the create items. We would lose that association if it were named `session_table_import_fail|success`. So I think the longer name is better in this case (which is not my usual stance!).

 It is not clear to me at the moment if it is a subset of the other. I think yes, but that answer could influence naming. That is, no matter what the naming, if the import fails and increments your new statistic, would `session_table_create_fail` also get incremented? If it does, it is probably important to make that association clear in the naming and description with something like "table create with import failed calls"

Reply:
Thank you for the suggestions. It does also seem to me that the statistic should be a subset of the other. Currently, the new statistic would be incremented if the `__schema_create` function fails and the import config is set. The existing `session_table_create_fail` statistic will also be incremented back in the `__session_create` function. 

Maybe we should call the new statistic `session_table_create_with_import_fail|success`?